src/Mobs/MagmaCube.cpp

```diff
@@ -38,3 +38,14 @@ AString cMagmaCube::GetSizeName(int a_Size)
 		return "small";
 	}
 }
+
+
+
+
+
+void cMagmaCube::WriteMetadata(cMetadataWriter & a_Writer) const
+{
+	super::WriteMetadata(a_Writer);
+	// Note: This is slime metadata
+	a_Writer.WriteInt(GetSize());  // Size
```

Comment:
If the Slime metadata would change, it would be changed in the `cSlime` class, but who would know, that the `cMagmaCube` also needs to be changed :confused:?